sql/2020/19_Compression/19_02.byte_saving_of_text_compression.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
```

Comment:
I'm unable to run this query as it says that it exceeds the execution limits.   Were you able to run it?

Reply:
me, either, would you gave some suggestion on how to fix 19_02. 
also seen that caching chapter has some similar metrics as ours, and think these metrics may be combined as one. here's the items and question hope hear your guidance. thanks.
(19_02) 20_Distribution of potential byte savings from the Lighthouse caching audit. 
(19_04) 20_Distribution of Lighthouse scores for the "Uses Long Cache TTL" audit for mobile web pages.  
Not sure that would be possible to join long cache TTL request with lighthouse scores.